README.md

```diff
@@ -0,0 +1,72 @@
+Diagnostic Log and Trace - README.md
+
+Overview
+---
+The Diagnostic Log and Trace Viewer is an application,
+written for Windows and Qt, that can send and receive control messages
```

Comment:
Please change to:

based on Qt for Windows and Linux (optional Mac)

We test DLT Viewer only for Windows and Linux, Mac is tested by some users from time ton time.